src/steps/command.ts

```diff
@@ -37,16 +49,19 @@ export class Command {
     toString(): string {
         return this.command;
     }
-}
 
-type Agents = Map<string, string>;
+    protected serialize(): string {
+        return this.toString();
+    }
+}
 
-const transformCommand = (value: Command[]): undefined | string | string[] => {
-    if (!value || value.length === 0) {
-        return undefined;
+export abstract class MetaCommand extends Command {
```

Comment:
this meta command is not needed, or is it? We can implement the Retry command w/o it I think?

Reply:
Moved this into examples

Reply:
Removed it entirely, it doesn't add much as an example